src/components/dialogs/create-study-dialog/create-study-dialog.js

```diff
@@ -126,6 +127,21 @@ const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
                         return parameter;
                     });
 
+                    const currentParameters = result.parameters.map(
+                        (param) => ({
+                            [param.name]: Array.isArray(param.defaultValue)
+                                ? param.defaultValue.toString()
+                                : param.defaultValue,
+                        })
+                    );
+
+                    const currentParametersObj = Object.assign(
+                        {},
+                        ...currentParameters
+                    );
+
+                    setValue(CURRENT_PARAMETERS, currentParametersObj);
```

Comment:
you can use reduce

const currentParametersObj = result.parameters.reduce((obj, param) => {
  obj[param.name] = Array.isArray(param.defaultValue) ? param.defaultValue.toString() : param.defaultValue;
  return obj;
}, {});

setValue(CURRENT_PARAMETERS, currentParametersObj);